index.js

```diff
@@ -71,11 +71,17 @@ function iconfontCSS(config) {
 			});
 		}
 
-		currentCodePoint = currentGlyph.toString(16).toUpperCase();
+		var fileName = path.basename(file.path, '.svg');
+		
+		if (config.fixedCodepoints && config.fixedCodepoints[fileName]){
+			currentCodePoint = config.fixedCodepoints[fileName].toString(16).toUpperCase();
+		} else {
+			currentCodePoint = (currentGlyph++).toString(16).toUpperCase();
```

Comment:
If I understand correctly, this would increase the `currentCodePoint` value pushed to `glyphMap` compared to the previous version.